src/main.js

```diff
@@ -677,10 +698,16 @@ function addEleContent(e) {
   var openWindow = window.open(newNodeAddress).document.body.innerHTML += appendElement;
 
   if (nodes.find(x => x.id === hoveredEle)) {
-    nodes.find(x => x.id === hoveredEle).content = true;
+    nodes.find(x => x.id === hoveredEle).content = "true";
+    if (document.getElementById(hoveredEle)) { //Prevent finding null to error out the entire page;
+      document.getElementById(hoveredEle).setAttribute("content","true");
```

Comment:
This is ok for now. Need to refactor for next iteration to separate data and model.